config/locales/views/circuitverse/he.yml

```
@@ -0,0 +1,154 @@
+        item5_text: "צור תת-מעגל פעם אחת והשתמש בו שוב ושוב, מה שמפשט את העיצוב והופך אותו למובנה יותר."
+        item6_heading: "אוטובוס ביטים מרובים ורכיבים"
+        item6_text: "CircuitVerse תומך בחוטים מרובי ביטים, מה שהופך את תכנון המעגלים לפשוט, מהיר ומסודר יותר."
+    featured_examples:
+        main_heading: "דוגמאות מומלצות"
+        main_description: "למד וקבל השראה מהדוגמאות הטובות ביותר"
+        circuit1_text: "מחבר ריפל קארי 16 ביט"
+        circuit2_text: "SAP-1"
+        circuit3_text: "מחבר מלא משני מחברים חציוניים"
+    explore_more: "חקור עוד"
+    editor_picks:
+        main_heading: "בחירות העורך"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)